src/Management/src/Endpoint/Actuators/Health/EndpointServiceCollectionExtensions.cs

```diff
@@ -60,7 +95,7 @@ public static IServiceCollection AddHealthContributor<T>(this IServiceCollection
     {
         ArgumentNullException.ThrowIfNull(services);
 
-        services.TryAddEnumerable(ServiceDescriptor.Scoped<IHealthContributor, T>());
+        services.TryAddEnumerable(ServiceDescriptor.Singleton<IHealthContributor, T>());
```

Comment:
Do you think there is anything to be concerned about with the change from scoped to singleton? How likely is it that a custom `IHealthContributor` would be negatively affected?

Reply:
I can't think of a scenario where this would be useful. However, if a health contributor needs it, an `IServiceProvider` can be injected to create a scope, or `IHttpContextAccessor` to obtain scoped services bound to the currently executing HTTP request.

Also, documentation at https://docs.steeltoe.io/api/v3/management/health.html mentions:
> The following example shows how to enable the `health` endpoint and add a custom `IHealthContributor` to the service container by adding `CustomHealthContributor` _as a singleton_.

which makes me think using scoped health contributors is not the recommended practice.